app/src/main/java/com/xmartlabs/daydreaming/ui/DashboardFragment.java

```diff
@@ -183,6 +186,20 @@ private boolean closeDrawer(@NonNull MenuItem menuItem) {
     return true;
   }
 
+  private void showRateAppDialog() {
+    new AlertDialog.Builder(getContext())
+        .setTitle(R.string.rate)
+        .setMessage(R.string.rate_app_now)
+        .setPositiveButton(R.string.rate_button, (dialog, which) -> openURL())
+        .setNegativeButton(R.string.close, (dialog, which) -> {})
+        .show();
+  }
+
+  private void openURL() {
+    Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(XMARTLABS_URL));
```

Comment:
Because there aren't any place right now that allow you to rate the app because there isn't upload anywhere yet. The purpose of that is only to show something.